extensions/2.0/Vendor/EXT_materials_clearcoat_darkening/schema/material.EXT_materials_clearcoat_darkening.schema.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "EXT_materials_clearcoat_darkening glTF Material Extension",
+    "type": "object",
+    "description": "glTF extension that defines the amount that the coat darkens the underlying surface due to internal reflections.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "clearcoatDarkeningFactor": {
+            "type": "array",
```

Comment:
If it's just one value, consider making it a `number` instead of an `array`.

Reply:
Done